client/interfaces.go

```diff
@@ -78,21 +78,17 @@ func (client *Client) Interfaces() (interfaces Interfaces, err error) {
 }
 
 // performInterfaceAction performs a single action on the interface system.
-func (client *Client) performInterfaceAction(sa *InterfaceAction) error {
+func (client *Client) performInterfaceAction(sa *InterfaceAction) (string, error) {
 	b, err := json.Marshal(sa)
 	if err != nil {
-		return err
+		return "", err
 	}
-	var rsp interface{}
-	if _, err := client.doSync("POST", "/v2/interfaces", nil, bytes.NewReader(b), &rsp); err != nil {
-		return err
-	}
-	return nil
+	return client.doAsync("POST", "/v2/interfaces", nil, bytes.NewReader(b))
 }
 
 // Connect establishes a connection between a plug and a slot.
 // The plug and the slot must have the same interface.
-func (client *Client) Connect(plugSnapName, plugName, slotSnapName, slotName string) error {
+func (client *Client) Connect(plugSnapName, plugName, slotSnapName, slotName string) (string, error) {
```

Comment:
Can we please name the results here and below, so docs tell easily what the first argument is?